src/renderer/components/ResourcesLinks/index.jsx

```diff
@@ -60,10 +62,12 @@ export default function ResourcesTab(props) {
   return (
     <React.Fragment>
       <a href={userGuideURL} onClick={handleClick}>
-        {"User's Guide"}
+        <MdOpenInNew />
+        {" User's Guide"}
       </a>
       <a href={forumURL} onClick={handleClick}>
-        Frequently Asked Questions
+        <MdOpenInNew />
+        {' Frequently Asked Questions'}

```

Comment:
Are the JSX brackets useful here to get that space in the string? Instead of using a space as a separator between the icon and text, I wonder if it'd be a better practice to add a CSS class to icons and use `margin-right` or `pad`?